pyproject.toml

```diff
@@ -23,9 +23,9 @@ python = "^3.9"
 datasketch = "*"
 e3fp = "*"
 joblib = "*"
-mhfp = "*"
 numpy = "^1.20.0"
 pandas = "*"
+pytest = "^8.1.1"
```

Comment:
Why do we have pytest dependency added twice? (Second in test group dependencies)

Reply:
Good catch, I removed it